docker/airflow/Dockerfile

```
@@ -1,17 +1,18 @@
 # Cannot use $PYTHON_VERSION because that is overwritten by the base image
 # with the patch version (we only care about major and minor version)
 ARG PROJECT_PY_VERSION
```

Minor suggestion for clarity:
```suggestion
ARG PROJECT_PYTHON_VERSION
```

---

We'd need to change this in a few places (I think in the CI/CD files files as well):

https://github.com/WordPress/openverse-catalog/blob/fba7566540343549b5a01211e69c9237c0ed6863/.github/workflows/ci_cd.yml#L83

If it's okay with you, I'll defer this for now and we can make the change in a separate PR 🙂